config/appdal/moduleconfs.data.xml

```diff
@@ -117,6 +128,20 @@
  <rel name="data_store_params" class="DataStoreConf" id="default"/>
 </obj>
 
+<obj class="HSI2TCTranslatorConf" id="hsi-to-tc-conf">
+ <attr name="template_for" type="class" val="DataReader"/>
+ <attr name="emulation_mode" type="bool" val="0"/>
+ <rel name="signals">
+  <ref class="HSISignalWindow" id="def-hsi-signal-window"/>
+ </rel>
+</obj>
+
+<obj class="HSISignalWindow" id="def-hsi-signal-window">
+ <attr name="signal_type" type="u32" val="1"/>
+ <attr name="time_before" type="u32" val="10000"/>
+ <attr name="time_after" type="u32" val="20000"/>
+</obj>
+
```

Comment:
Not for this PR, but we will need to think about HSI a little more in the future for v5.
There were large changes to how we deal with them in v4 because of the new CTB, CIB and PNS signals that are also going through HSIEvents but to separate modules, and even the standard `TimingTriggerCandidateMaker` configuration was changed substantially for some of that.